tests/usm/usm_api.h

```diff
@@ -217,11 +217,35 @@ void check_values(const T *begin, const ReferenceIt &reference) {
   }
 }
 
+/** @brief Trait for tests that have no additional device requirements
+ *
+ * Tests implementing this trait do not require any device support beyond
+ * the destination memory location.
+ */
+struct noAdditionalDeviceRequirements {
+  static bool supports_device(sycl_cts::util::logger& log,
+                              const sycl::queue& queue) {
+    return true;
+  }
+};
```

Comment:
```suggestion
  static bool supports_device(sycl_cts::util::logger&,
                              const sycl::queue&) {
    return true;
  }
};

```
to avoid warning about unused arguments and more spacing between function definitions.

Reply:
Ah, using Github's resolve thing doesn't use the correct email. One sec.